maintenance/main.py

```diff
@@ -177,6 +183,10 @@ def black_format(code):
 
 
 def ensembl_stdpopsim_id(ensembl_id):
+    if ensembl_id == "canis_lupus_familiaris":
```

Comment:
perhaps insert a comment explaining what this is doing?

Reply:
I mean, explaining why this is necessary

Reply:
Okay - could you either take this out, or say in the comment exactly what needs to happen to take it out? (It says "once we have moved to the new names" - but: moved what names? where? I am confused. =)

Reply:
added a larger comment now explaining the context that says:

```python
    # below is do deal with name changes in Ensembl
    # TODO: remove this once we have moved to the new names
    # this was added as a temporary fix to allow the release to be
    # updated to 113, where the names of these species were changed
    # in the future we might keep this code block, but comment it out
    # to show others how maintenance updates were performed in the case
    # where the species name changed in Ensembl
```

Reply:
This isn't what I understood from talking to you - I thought that (a) we could remove this code right now if we wanted, since this `if` clause will never get triggered; but (b) we were going to leave the code in as an example of how to do this. Is that right?